eterm-256color.el

```diff
@@ -40,6 +40,13 @@
   "256colors for term."
   :group 'term)
 
+(defcustom eterm-256color-terminfo-path (pcase system-type
```

Comment:
Could you add `:group 'eterm-256color` and `:type 'string` to this?

Reply:
Done.